ExtraTests.cmake

```diff
@@ -0,0 +1,96 @@
+if(BUILD_TESTING)
+  set(EXCL_ALL)
+else()
+  set(EXCL_ALL EXCLUDE_FROM_ALL)
+endif()
+
+set(TEST_LIBS opmparser ecl Boost::unit_test_framework)
+
+# Generated source, needs to be here
+add_executable(inlinekw ${EXCL_ALL} ${PROJECT_BINARY_DIR}/inlinekw.cpp)
+target_link_libraries(inlinekw ${TEST_LIBS})
+add_test(NAME InlineKeywordTest COMMAND inlinekw)
```

Comment:
probably, using `opm_add_test()` here is a good idea. this way `make test-suite` will work properly...

Reply:
indeed. no idea why i didn't do that instead of the half-arsed reimplementation. remedied.